src/expression/embeddedDocs/function/statistics/cumsum.js

```diff
@@ -0,0 +1,24 @@
+export const cumSumDocs = {
+  name: 'cumsum',
+  category: 'Statistics',
+  syntax: [
+    'cumsum(a, b, c, ...)',
+    'cumsum(A)'
+  ],
+  description: 'Compute the sum of all values.',
+  examples: [
+    'cumsum(2, 3, 4, 1)',
+    'cumsum([2, 3, 4, 1])',
+    'cumsum([2, 5; 4, 3])'
```

Comment:
I got this from sum.js in the same folder. Should the semicolon be there?

Reply:
@josdejong

Reply:
The semicolon makes it a 2x2 matrix.